lms/static/js/edxnotes/views/search_box.js

```diff
@@ -10,7 +10,7 @@ define([
         },
 
         errorMessage: gettext('An error has occurred. Make sure that you are connected to the Internet, and then try refreshing the page.'),
-        emptyFieldMessage: gettext('Please enter a term in the search field.'),
+        emptyFieldMessage: gettext('Please enter a term in the <a href="#search-notes-input">search field</a>.'),
```

Comment:
@polesye the gettext call looks a lot better the way you suggest. I don't know JS very well so I can't comment as to its correctness.

@talbs fyi